cli/types/index.d.ts

```diff
@@ -29,7 +29,7 @@ declare const assert: Chai.AssertStatic
 declare namespace Cypress {
   type FileContents = string | any[] | object
   type HistoryDirection = "back" | "forward"
-  type HttpMethod = "GET" | "POST" | "PUT" | "DELETE" | "OPTIONS" | "HEAD" | "TRACE" | "CONNECT" | "PATCH"
+  type HttpMethod = "GET" | "POST" | "PUT" | "DELETE" | "PATCH" | "OPTIONS" | "HEAD" | "TRACE" | "COPY" | "LOCK" | "MKCOL" | "MOVE" | "PURGE" | "PROPFIND" | "PROPPATCH" | "UNLOCK" | "REPORT" | "MKACTIVITY" | "CHECKOUT" | "MERGE" | "M-SEARCH" | "NOTIFY" | "SUBSCRIBE" | "UNSUBSCRIBE" | "SEARCH" | "CONNECT"

```

Comment:
I wonder if this should just be `string`. From what I can tell, node's type definition just use `string`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/v8/index.d.ts#L1032

Reply:
Technically speaking, this list of methods could change with any new Node.js release. We also accept mixed casing like you've noted, so technically this list of valid method names would be a looooooot longer if it encompassed everything.

I think `string` would be fine here.